### ISSUE TITLE
Request: Make it easier for end users to add servers

### ISSUE THREAD

**Issue by KronK0321** [benediktschmitt/emsm#97]
Currently, it is not technically feasible for end users to add servers. 

Simply editing server.conf only allows for overriding existing server download URLs. 

It appears the various server options are hardcoded within the core/server.py file as classes. 

It would be nice if users were able dynamically add servers so we are not at the mercy of the emsm package being up-to-date.

**Comment by bryceharrington** [comment]
Agreed, although admittedly it's possible to edit server.py and add versions or update jars manually.  Good way to start making contributions to the project I suppose.  (And if more people were actively doing this, the problem would diminish.)

That said, I agree it would be nice to have a simple config file allowing adding new minecraft versions or overriding the jars provided by an existing version.  If nothing else it'd make testing new versions of jars a bit less involved.
